### PR TITLE
Modification to Config Map test cases

### PR DESCRIPTION
**Pull request by msvinaykumar**
Signed-off-by: msvinaykumar <vinakuma@redhat.com>

Test cases are modified to handle #370   changes. Modification to Config Map test cases

### CODE REVIEW

**Comment by chandrams** [comment]
@msvinaykumar - We will need to update other tests too that use autotune log messages. Also in configmap tests a new test needs to be included for root_logging_level

File: tests/scripts/da/constants/configmap_constants.sh [hunk 1]
```diff
@@ -30,37 +30,43 @@ configmap_minikube_testcases=("minikube-invalid-cluster-type"
...
 # Expected log message for minikube configmap
 declare -A configmap_minikube_expected_log_msgs
-deployment_info='com.autotune.analyzer.deployment.DeploymentInfo'
-configmap_minikube_expected_log_msgs=([minikube-invalid-cluster-type]=''${deployment_info}' - Cluster type kube8 is not supported'
-[minikube-invalid-k8s-type]=''${deployment_info}' - k8s type docker is not suppported'
-[minikube-invalid-monitoring-agent]=''${deployment_info}' - Monitoring agent promo  is not supported'
-[minikube-invalid-monitoring-service]='com.autotune.analyzer.datasource.DataSourceFactory - Monitoring agent endpoint not found'
-[minikube-debug-config]='DEBUG' )
+deployment_info='[AutotuneDeploymentInfo.java([0-9]*)]'
+configmap_minikube_expected_log_msgs=([minikube-invalid-cluster-type]=''${deployment_info}'-Cluster type kube8 is not supported'
```
Comment (chandrams) [inline tests/scripts/da/constants/configmap_constants.sh:67]: Why is the space removed after hyphen?

Comment (msvinaykumar) [reply tests/scripts/da/constants/configmap_constants.sh:67]:
```
2022-01-2105:59:25.188 INFO [OkHttp https://10.96.0.1/...][AutotuneDeployment.java(157)]-Added autotuneconfig container
2022-01-2105:59:25.188 INFO [OkHttp https://10.96.0.1/...][AutotuneDeployment.java(157)]-Added autotuneconfig container
```
Hello @chandrams 
New loghandler format prints message without space and hence space is removed. Please let me know if this is ok?

**Comment by chandrams** [comment]
@msvinaykumar - You need to update autotune_config_yaml this test as well as messages there too get affected

**Comment by chandrams** [comment]
app_autotune_yaml tests too might need changes and all messages that are affected needs to be fixed:

autotune_config_obj_create_msg='com.autotune.analyzer.deployment.AutotuneDeployment - Added autotuneconfig'
exception="com.autotune.analyzer.exceptions.InvalidValueException:"
invalid_bound_exception='com.autotune.analyzer.exceptions.InvalidBoundsException'

**Comment by msvinaykumar** [comment]
> exception="com.autotune.analyzer.exceptions.InvalidValueException:"
> invalid_bound_exception='com.autotune.analyzer.exceptions.InvalidBoundsException'

no change required as of now as it is printing fully qualified name.

**Comment by msvinaykumar** [comment]
> autotune_config_obj_create_msg='com.autotune.analyzer.deployment.AutotuneDeployment - Added autotuneconfig'

change done please review

**Comment by msvinaykumar** [comment]
@chandrams  Please review

File: tests/scripts/da/constants/app_autotune_yaml_constants.sh [hunk 2]
```diff
@@ -205,8 +205,8 @@ slo_class_autotune_objects=([blank-sloclass]='true'
 declare -A slo_class_expected_log_msgs
 yaml_test_path="${path}/${app_autotune_tests[1]}"
 slo_kubectl_error=': error validating data: ValidationError(Autotune.spec): missing required field "slo" in com.recommender.v1.Autotune.spec; if you choose to ignore these errors, turn validation off with --validate=false'
-slo_class_expected_log_msgs=([blank-sloclass]=''${autotune_object_create_msg}' blank-sloclass' 
-[invalid-sloclass]=''${autotune_exception}' slo_class: rgyedg not supported' 
+slo_class_expected_log_msgs=([blank-sloclass]=''${autotune_object_create_msg}' blank-sloclass'
+[invalid-sloclass]=''${autotune_exception}' sla_class: rgyedg not supported'
```
[the remark this answers is not in the document]
Comment (msvinaykumar) [reply tests/scripts/da/constants/app_autotune_yaml_constants.sh]: done

**Review by chandrams — APPROVED** [review]
LGTM

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**